server/src/main/java/no/ntnu/klubbhuset/resource/OrganizationResource.java

```diff
@@ -87,4 +90,11 @@ public Response joinOrganization(@PathParam("organizationId") Long id) {
     public Response getMembers(@PathParam("organizationId") String organizationId) {
         return organizationService.getMembers(organizationId);
     }
+
+    @GET
+    @Path("/owned")
```

Comment:
Should this maybe be "/managed" if this endpoint is meant for retrieving organizations that this user is managing?